src/index.html

```diff
@@ -10,13 +10,104 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
```

Comment:
The `meta` tag for 'X-UA-Compatible' also exceeds the 80 characters per line limit. It should be split into multiple lines.